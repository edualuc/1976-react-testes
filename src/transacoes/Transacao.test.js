import { render, renderer } from "@testing-library/react"
import React from "react";

import Transacao from "./Transacao";

describe('Componte de transação do extrato', () => {
  it('O snapshot do componente deve permanacer o sempre o mesmo', () => {
    const { container } = render(<Transacao data="10/08/2020" tipo="saque" valor="20.00" />);
    expect(container.firstChild).toMatchSnapshot();
  });
  // it('O snapshot do componente deve permanacer o sempre o mesmo2', () => {
  //   const transacaoComponent = renderer.create(
  //     <Transacao data="10/08/2020" tipo="saque" valor="20.00" />,
  //   );
  //   const transacaoTree = transacaoComponent.toJSON();
  //   expect(transacaoTree).toMatchSnapshot();
  // });
})