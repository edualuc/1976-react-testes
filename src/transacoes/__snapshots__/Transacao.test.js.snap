// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Componte de transação do extrato O snapshot do componente deve permanacer o sempre o mesmo 1`] = `
<div
  class="transacao-container"
>
  <p>
    10/08/2020
  </p>
  <p>
    saque
  </p>
  <p>
    R$ 
    20.00
  </p>
</div>
`;
